### ISSUE TITLE
Confirmed works with VR

### ISSUE THREAD

**Issue by tb5926** [Santi871/DCSWaypointEditor#52]
I successfully utilized this with VR active.  I followed all instructions given in the 'How To' with success.  I was able to navigate to the 'data' folder that is unzipped with the program.  I opened the PG.JSON, saved as and edited with my own desired preset waypoints that populated in the waypoint editor program.  I did this as they are native to google maps output.  It was a simple copy paste.  I did not change the F18 coordinate type and left at its default upon start-up.  After coordinates were entered I swapped between DMS and DDS with 100% accuracy from the waypoint editor program.  Thank you @Santi871 and other contributors for making this.  My next test is to build a flight in combat flight and export the .json file to the 'data' folder location.

**Comment by Jeffb4u** [comment]
Yes i can also confirm this part works (using the stored co-ords in data worked for me and go to the pane no issues.
- Capturing from the map does not seem to work for me though - reinstalled re-read instructions 
- still states "Failed to Capture" below Capture from F10 Button.
I would love this to work as it is an absolutely brilliant bit of kit. but sadly.......
Plus no response from the coders so likely no longer supported

**Comment by Jeffb4u** [comment]
Ahhhh Solved it fully works for all -- im off to make sequences lovely sequences with zero typing

**Comment by e4mafia** [comment]
What did you do to fix the F10 failed to capture bit?

**Comment by Jeffb4u** [comment]
OK so note i wasnt talking about getting it to work in VR just on the monitor using track IR - My friend has VR and he cant get his headset to focus on his desktop so dont know how the VR bit works.

However, the issue for me was i had nvidia performance data FPS as an overlay in the top left corner so when it took an image the OCR could not read it, when i turned that off it worked fine.
I recall i had added some debug lines in the settings file to log what was happening and store an image of the screen capture and when i looked and saw my FPS it was clear why it wasnt working.  These debug text lines in that file were from another issue thread with the app downloaded direct from there i cant recall his name but he is linked to the project.
So i got it working with that version initially. 0.9.9b3 test ish or something i think.
But i later DL 1.0.0b1 or the latest version listed (can be hard to find oddly), this works the same as the test version but has the predefined waypoints listed and working.
(sadly this list is in alphabetical order over the 3 maps so Batumi is a long way down the list from Anapa after all the al daf...s... etc.  could have been split cauc, persian, but hey ho)

Then the only issue with the latest version is when you run it, it says there is a newer version, but that installs to 0.9.9b2 if you say OK so say No.

Other than that it can take a while to work out how things work, the Ctrl,shift.T and Ctrl.T seem counter intuitive to the operating instructions.

So it works but is finicky to get used to - but it is amazing watching all the cockpit buttons doing their thing.

I think it would be most useful in the F18 as other aircraft have inbuilt systems for collecting/creating waypoints, like harrier A10C.

Maybe thats why this has had no input for a while, but it could be useful in some mission scenarios.

**Comment by Jeffb4u** [comment]
Ive looked for the settings text you need to add to get the images to store.

[PREFERENCES]
grace_period = 5
button_release_short_delay = 0.2
button_release_medium_delay = 0.5
tesseract_path = D:/Data/Applications/Tesseract/tesseract.exe
dcs_path = C:/Users/Graig/Saved Games/DCS.openbeta/
db_name = profiles_new.db
capture_key = 1
quick_capture_hotkey = ctrl+shift+t
enter_aircraft_hotkey =
log_raw_tesseract_output = false
save_debug_images = false


add these last 2 lines and i guess set them to true

also see issue #48 then follow that to #32 to get the test version from "Rhulsker" - confirm it works with this then you should have the knowledge to install and use the latest version (remembering not to update it)

Enjoy, hope this helps
